package.json

```diff
@@ -6,7 +6,7 @@
   "main": "app.js",
   "scripts": {
     "start": "node app.js",
-    "dev-start": "source ./env/development && source ./env/dev-secrets && node --debug app.js",
+    "dev-start": "source ./env/development && source ./env/dev-secrets && node --inspect app.js",
```

Comment:
Wasn't working on mine either. Wanted to start a PR so we're tracking it as a todo